go.sum

```diff
@@ -23,6 +23,8 @@ github.com/oklog/ulid/v2 v2.1.0/go.mod h1:rcEKHmBBKfef9DhnvX7y1HZBYxjXb0cP5ExxNs
 github.com/pborman/getopt v0.0.0-20170112200414-7148bc3a4c30/go.mod h1:85jBQOZwpVEaDAr341tbn15RS4fCAsIst0qp7i8ex1o=
 github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
+github.com/riverqueue/river/riverdriver/riverpgxv5 v0.0.4 h1:336o1gbjpsSe1I9Es0eoJkBagXTmkXdzZ1M8j9kwLUU=
+github.com/riverqueue/river/riverdriver/riverpgxv5 v0.0.4/go.mod h1:k6hsPkW9Fl3qURzyLHbvxUCqWDpit0WrZ3oEaKezD3E=
```

Comment:
1. What's the process for cutting a new release if we go this route?
2. As part of cutting a new release, will the `go.mod` and `go.sum` disagree with each other around the required version of `riverpgxv5`?
3. After shipping this change, if a user installs `river@latest`, will they also get the latest `riverpgxv5` no matter what this `go.sum` says?

Reply:
> 1. What's the process for cutting a new release if we go this route?

I think either:

1. Use the same release process as before and accept that the build is temporarily broken.
2. Two phase release where river is tagged and after that's in, riverpgxv5 is tagged.

> 2. As part of cutting a new release, will the `go.mod` and `go.sum` disagree with each other around the required version of `riverpgxv5`?

I don't know.

> 3. After shipping this change, if a user installs `river@latest`, will they also get the latest `riverpgxv5` no matter what this `go.sum` says?

I think the `go.sum` is only locking for the current module, but not 100% sure about this.